configs/WDIYH743/config.h

```diff
@@ -0,0 +1,146 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32H743
+
+#define BOARD_NAME        WDIYH743
+#define MANUFACTURER_ID   WDIY
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_FLASH_W25N01G
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_MAX7456
+ 
+#define BEEPER_PIN           PA10
+#define MOTOR1_PIN           PA0
+#define MOTOR2_PIN           PA1
+#define MOTOR3_PIN           PA2
+#define MOTOR4_PIN           PA3
+#define MOTOR5_PIN           PB4
+#define MOTOR6_PIN           PB5
+#define MOTOR7_PIN           PB0
+#define MOTOR8_PIN           PB1
+#define SERVO1_PIN           PE11
+#define SERVO2_PIN           PE14
+
+#define RX_PPM_PIN           PC6
+#define LED_STRIP_PIN        PB8
+
+#define UART1_TX_PIN         PA9
+#define UART2_TX_PIN         PD5
+#define UART3_TX_PIN         PD8
+#define UART4_TX_PIN         PB9
+#define UART6_TX_PIN         PC6
+#define UART7_TX_PIN         PE8
+#define UART8_TX_PIN         PE1
+#define UART1_RX_PIN         PB15
+#define UART2_RX_PIN         PD6
+#define UART3_RX_PIN         PD9
+#define UART4_RX_PIN         PB8
+#define UART6_RX_PIN         PC7
+#define UART7_RX_PIN         PE7
+#define UART8_RX_PIN         PE0
+
+#define I2C1_SCL_PIN         PB6
+#define I2C2_SCL_PIN         PB10
+#define I2C3_SCL_PIN         PA8
+#define I2C4_SCL_PIN         PD12
+#define I2C1_SDA_PIN         PB7
+#define I2C2_SDA_PIN         PB11
+#define I2C3_SDA_PIN         PC9
+#define I2C4_SDA_PIN         PD13
+
+#define LED0_PIN             PE12
+#define LED1_PIN             PE13
+ 
+#define SPI1_SCK_PIN         PA5
+#define SPI1_SDI_PIN         PA6
+#define SPI1_SDO_PIN         PA7
+
+#define SPI2_SCK_PIN         PB13
+#define SPI2_SDI_PIN         PC2
+#define SPI2_SDO_PIN         PC3
+ 
+#define SPI3_SCK_PIN         PB3
+#define SPI3_SDI_PIN         PC11
+#define SPI3_SDO_PIN         PB2
+
+#define SPI4_SCK_PIN         PE2
+#define SPI4_SDI_PIN         PE5
+#define SPI4_SDO_PIN         PE6
+
+#define ADC_VBAT_PIN         PC0
+#define ADC_RSSI_PIN         PC4
+#define ADC_CURR_PIN         PC1
+
+#define FLASH_CS_PIN         PA15
+#define MAX7456_SPI_CS_PIN   PB12
+#define GYRO_1_EXTI_PIN      PE9
+#define GYRO_2_EXTI_PIN      PE3
+#define GYRO_1_CS_PIN        PA4
+#define GYRO_2_CS_PIN        PE4
+ 
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 1, PA1,  2,  0) \
+    TIMER_PIN_MAP( 2, PA2,  2,  1) \
+    TIMER_PIN_MAP( 3, PA3,  2,  2) \
+    TIMER_PIN_MAP( 4, PB4,  2,  3) \
+    TIMER_PIN_MAP( 5, PB5 , 2,  4) \
```

Comment:
Timer index 2 is invalid
```suggestion
    TIMER_PIN_MAP( 4, PB4,  1,  3) \
    TIMER_PIN_MAP( 5, PB5 , 1,  4) \
```